src/components/Inputs/Technoeconomic/TechnoeconomicInputs.tsx

```diff
@@ -25,7 +19,34 @@ interface Props {
   disabled: boolean;
 }
 
+const determineScaledCapitalCost = (electricalCapacity: number) => {
+  const scaleFactor = 0.82; // source: https://evcvaluation.com/cost-to-capacity-method-applications-and-considerations/
+  const defaultCapitalCost = 70_000_000;
+  const defaultElectricCapacity = 25_000;
+
+  if (electricalCapacity === defaultElectricCapacity) {
+    return defaultCapitalCost;
+  } else {
+    const scaleCapitalCost =
+      (defaultCapitalCost * (electricalCapacity / defaultElectricCapacity)) ^
+      scaleFactor;
+    return scaleCapitalCost;
+  }
```

Comment:
should first `(electricalCapacity/defaultElectricCapacity)**scaleFactor` and then multiply it by `defaultCapitalCost`
this can be written as one liner `return defaultCapitalCost * (electricalCapacity/defaultElectricCapacity)**scaleFactor;`
because if `electricalCapacity === defaultElectricCapacity`, `electricalCapacity / defaultElectricCapacity = 1` and `defaultCapitalCost * (electricalCapacity/defaultElectricCapacity)**scaleFactor = defaultCapitalCost`.

Reply:
@kaiyanl ok good one -- i just updated the code see if that's better.  Now it is `const scaleCapitalCost = defaultCapitalCost * (electricalCapacity/defaultElectricCapacity)^scaleFactor `